### PR TITLE
Simplify examples

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: example/tree_shake.dart [hunk 1]
```diff
@@ -1,42 +0,0 @@
-// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
```
Comment (lrhn) [inline example/tree_shake.dart]: I use these files in `tool/check_tree_shaking.dart`, so they should be moved below `tool/src/` somewhere, probably `tool/src/tree_shaking/*.dart` (and update the `check_tree_shaking.dart` file to point there).

I can do that if you don't delete the files.

File: example/platform_using_switch.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import 'package:platform/platform.dart';
+
+void main() {
+  switch (Platform.current) {
+    case Platform(:var browserPlatform?):
+      print('Running in a browser');
+      print('User-agent: ${browserPlatform.userAgent}');
+    case Platform(nativePlatform: var platform?)
+        when platform.isLinux | platform.isMacOS | platform.isWindows:
```
Comment (lrhn) [inline example/platform_using_switch.dart:13]: (Should probably check that this pattern is tree-shaken correctly. It *probably* is, I've checked with just, fx, `when platform.isLinux`.)

**Review by lrhn — APPROVED** [review]
Leave the `tree_shake_*.dart` files, otherwise OK.

**Comment by mit-mit** [comment]
replaced by https://github.com/dart-lang/platform/pull/52

**Pull request closed without merging**